File: QubitServer/src/org/labrad/qubits/channeldata/AnalogData.java
package org.labrad.qubits.channeldata;

import org.labrad.qubits.channels.AnalogChannel;


public interface AnalogData extends Deconvolvable {
  public void setChannel(AnalogChannel channel);
  public int[] getDeconvolved();
  public void checkLength(int expected);
}

File: QubitServer/src/main/scala/org/labrad/qubits/channeldata/AnalogData.scala
package org.labrad.qubits.channeldata

import org.labrad.qubits.channels.AnalogChannel

trait AnalogData extends Deconvolvable {
  def setChannel(channel: AnalogChannel): Unit
  def getDeconvolved(): Array[Int]
  def checkLength(expected: Int): Unit
}
